src/mpid/ch4/shm/ipc/src/ipc_fd.c

```diff
@@ -101,11 +282,6 @@ int MPIDI_IPC_mpi_fd_recv(int rank, int *fd, void *payload, size_t payload_len,
                                                 (msg.msg_flags & MSG_CTRUNC));
     MPIR_ERR_CHKANDJUMP(trunc_check, mpi_errno, MPI_ERR_OTHER, "**truncate");
 
-    if ((flags & MSG_PEEK) == 0 && ((msg.msg_flags & MSG_TRUNC) || (msg.msg_flags & MSG_CTRUNC))) {
-        mpi_errno = MPI_ERR_OTHER;
-        goto fn_fail;
-    }
-
```

Comment:
Why did we have this in the previous commit?

Reply:
It seems to be a duplicate of the above check of `trunc_check`. I've removed it from the previous commit.